lib/grammar.js

```diff
@@ -409,6 +409,31 @@ var grammar = module.exports = {
       reg: /^max-message-size:(\d+)$/,
       format: 'max-message-size:%s'
     },
+    {
+      // RFC7273
+      // a=ts-refclk:ptp=IEEE1588-2008:39-A7-94-FF-FE-07-CB-D0:37
+      push:'tsRefClocks',
+      reg: /^ts-refclk:([^\s=]*)(?:=(\S*))?/,
+      names: ['clksrc', 'clksrcExt'],
+      format: function (o) {
+        return 'ts-refclk:%s' + (o.clksrcExt !== undefined ? '=%s' : '');
+      }
+    },
+    {
+      // RFC7273
+      // a=mediaclk:direct=963214424
+      name:'mediaClk',
+      reg: /^mediaclk:(?:id=(\S*))? *([^\s=]*)(?:=(\S*))?(?: *rate=(\d+)\/(\d+))?/,
+      names: ['id', 'mediaClockName', 'mediaClockValue', 'rateNumerator', 'rateDenominator'],
+      format: function (o) {
+        var str = 'mediaclk:';
+        str += (o.id !== undefined ? 'id=%s %s' : '%v%s');
+        str += (o.mediaClockValue !== undefined ? '=%s' : '');
+        str += (o.rateNumerator !== undefined ? ' rate=%s' : '');
+        str += (o.rateDenominator !== undefined ? '/%s' : '');
```

Comment:
any reason you're consistently distinguishing between `null` and `undefined`? it's not a big thing, just means that users would have to delete the key rather than null it out to get the expected removal when serializing.